### ISSUE TITLE
Error while executing an SQL-query with 'round' function from aggregate

### ISSUE THREAD

**Issue by msiomkin** [tarantool/tarantool#5390]
Well, while [the previous](https://github.com/tarantool/tarantool/issues/5387) bug is fixed in 2.4, rounding an aggregate result still doesn't work in 2.4 and all newer versions (up to 2.6).

Tarantool version: Tarantool Enterprise 2.4.2-12-g11f587c

OS version: Centos 8

Bug description: error while executing an SQL-query with 'round' function from aggregate

Steps to reproduce:

```Lua
box.schema.create_space("test")

box.space.test:format({
    { name = 'id', type = 'unsigned' },
    { name = 'a', type = 'number' },
    { name = 'b', type = 'number' },
})

box.space.test:create_index("primary", { parts = { "id" } })

box.space.test:insert({ 1, 1, 1 })

box.execute([[
    select round(max("b"))
    from "test"
    group by "a"
    order by 1
    limit 1;
]])
```

The result:

```
---
- null
- 'Tuple field 1 type does not match one required by operation: expected integer'
...
```

**Comment by pgulutzan** [comment]
In Tarantool 2.10 the message is 'Type mismatch: can not convert number(1) to double'.
But that message happens even for simple queries e.g. SELECT round(CAST(1 AS NUMBER));
which, if I have understood correctly, is a deliberate behaviour change.

**Comment by ImeevMA** [comment]
Fixed by 0f144350a.
